### ISSUE TITLE
Add delay to redirect

### ISSUE THREAD

**Issue by santhosh-rajamani** [wi3land/ionic-appauth#199]
When the user enters the credentials in OKTA login page, it redirects immediately upon successful login to redirect_uri. It leaves no room for the user to save the password in browser if required. 

It would be better if we could configure a delay before redirect.

**Comment by nstekovic** [comment]
Having the same issue using angular capacitor, any workaround?

**Comment by mraible** [comment]
I don't think we can do anything in this library since the redirect is happening on the Okta side. You might try asking on the [Okta developer forums](https://devforum.okta.com).
